examples/whylabs/log_df_with_api_key.py

```diff
@@ -0,0 +1,30 @@
+"""
+Log a dataframe to WhyLabs
+===============
+
+Example of logging a dataframe with log rotation to WhyLabs
+"""
+import logging
+import os
+import time
+
+import pandas as pd
+
+from whylogs.app.session import get_or_create_session
+
+os.environ["WHYLABS_API_KEY"] = "<API-KEY>"
+os.environ["WHYLABS_API_ENDPOINT"] = "<end point override. not required>"
+
+module_logger = logging.getLogger()
+logging.basicConfig(level=logging.DEBUG)
+
+# Load some example data
+df = pd.read_csv("../data/lending_club_1000.csv")
+
+# Create a whylogs logging session
+session = get_or_create_session()
+# Log statistics for the dataset
+with session.logger(tags={"orgId": "org-2", "datasetId": "model-0"}, with_rotation_time="1s") as ylog:
```

Comment:
for the example script, we should probably replace `org-2` and `model-0` with `<YOUR_ORG_ID>` and `<YOUR_MODEL_ID>` or whatever, since these are unique to the user.

Reply:
Or specify `WHYLABS_DEFAULT_ORG_ID` and `WHYLABS_DEFAULT_DATASET_ID` with the other env vars above, and skip specifying tags on this line

Reply:
Updated